web/views.py

```
@@ -3391,8 +3391,11 @@ class GoodsDetailView(generic.DetailView):
     template_name = "goods/goods_detail.html"
     context_object_name = "product"
 
-    def get_object(self):
-        return get_object_or_404(Goods, pk=self.kwargs["pk"])
+    def get_context_data(self, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_context_data`

(ANN201)